### PR TITLE
 Added Postfix To Infix in C and C++

### LINKED ISSUE #978: Postfix to Infix expression  in C and c++

### PR DESCRIPTION
**Pull request by somya-kapoor**
Fixes #978 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

-  Added Postfix To Infix in C and C++

#### Languages Used:
- C
- C++

#### Files Added:
- Postfix_To_Infix.c
- Postfix_To_Infix.cpp

### CODE REVIEW

**Comment by MastersAbh** [comment]
Hey, good work with the topic, but please re-code it as it is exact similar to the code available at [https://www.codingalpha.com/postfix-to-infix-conversion-c-program/](url). Also, this is the second time your code has been found similar. please refrain from this. @jainaman224

**Comment by somya-kapoor** [comment]
@MastersAbh I haven't copied the code .I have used algorithm which was given in geeks for geeks .

**Comment by MastersAbh** [comment]
Hey @somya-kapoor ,
Your variable names, function calls and even length for stack array (20) is similar to the one available at the link, which makes it 95% similar. It is good if you have studied and implemented, but your code is way too similar right now. Please refrain and recode this.

**Comment by somya-kapoor** [comment]
@MastersAbh I have tried to write the code again. Please review

**Comment by MastersAbh** [comment]
It is not about using loops. The loop you are talking about was just initializing to zero, which could be well removed. Learning is encouraged, so please refrain.

**Comment by MastersAbh** [comment]
Please add comments explaining the logic in the code.

**Comment by somya-kapoor** [comment]
@MastersAbh I have added the comments. Please review.

File: Postfix_To_Infix/Postfix_To_Infix.c [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+{
+    return stack[top--];//Pop the top value from the stack.
+}
+
+void push(char ch)
+{
+    stack[++top] = ch;//Push it onto the stack.
+}
+
+int operand(char x)//Checking if it is operator or letter
```
Comment (amaaniqbal) [inline Postfix_To_Infix/Postfix_To_Infix.c]: Better to have return type as `bool` if you are returning a boolean value.

Also, have the comments about the function, above its definition for better readability. Try following this with other general comments too.

Comment (somya-kapoor) [reply Postfix_To_Infix/Postfix_To_Infix.c]: @amaaniqbal I have done the required changes.Please review

File: Postfix_To_Infix/Postfix_To_Infix.c [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        else
+        {
+            //pop the element present in the stack
+            element = pop(); 
+            //character is assigned to operator
+            operator = postfix[count];
+            printf(" %c %c", operator, element);            
+        }
+    }
+
```
Comment (amaaniqbal) [inline Postfix_To_Infix/Postfix_To_Infix.c]: Remove this blank line. Rest is fine. Very well done!

**Comment by somya-kapoor** [comment]
@amaaniqbal I have done the required changes . Please review.

**Pull request merged**